### ISSUE TITLE
Provide return_to value as part of flow entity

### ISSUE THREAD

**Issue by davidthor** [ory/kratos#1121]
**Is your feature request related to a problem? Please describe.**

When prompted to login, my users are given the option to signup instead by clicking a link under the login form. However, when this new registration flow is created, the return_to address is lost. When completing this new registration flow, the user will not be routed back to the return_to address provided when the login flow was first initialized. 

**Describe the solution you'd like**

It'd be great if the return_to address was included in the flow object returned by `GET /self-service/login/flows?id=string`. If it was part of this payload, I could append the `return_to` value to the signup link like this: `/self-service/registration/browser?return_to=${login_flow.return_to}`. This allows me to leverage Kratos for persisting this address and pass it to related flows without having to store it in a new session/cookie myself. 

**Describe alternatives you've considered**

Right now I'm storing the return_to URL in the users session and including it whenever the user clicks a link that would create a different flow path (e.g. password reset, registration, etc). While this works fine, its the only reason I need to create a session for some of these flows and it seems redundant with the data already stored in Kratos associated with each flow.

**Comment by zepatrik** [comment]
Right, makes sense :+1:

**Comment by aeneasr** [comment]
Absolutely! If you're up for a contribution I can help with some pointers :)

**Comment by davidthor** [comment]
> Absolutely! If you're up for a contribution I can help with some pointers :)

Some pointers would be helpful. I started browsing around the database schemas and didn't see where the return_to URLs are stored. Are those persisted in a session instead of being in the database alongside the rest of the flow object?

**Comment by aeneasr** [comment]
The information you desire is already included in https://github.com/ory/kratos/blob/master/selfservice/flow/login/flow.go#L56 IIRC. So basically you would take [that value](https://github.com/ory/kratos/blob/master/selfservice/flow/login/flow.go#L99), extract `return_to` and add it to the struct, which means that you have to add an additional field to the SQL table. To do that you need to create a new migration similar [to this one](https://github.com/ory/kratos/blob/master/persistence/sql/migrations/templates/20200830130642_add_verification_methods.up.fizz), and then run [`make migrations-render-replace`](https://github.com/ory/kratos/blob/master/Makefile#L144-L145).

And then of course add the field to existing tests and create a new test to check if the behaviour is correct!

**Comment by davidthor** [comment]
Thanks @aeneasr. I may have time to take a look this weekend.

**Comment by aeneasr** [comment]
🎉 🎉 🎉 🎉

**Comment by aeneasr** [comment]
Marked as stale in error.
